assets/app/views/builds.html

```diff
@@ -17,72 +17,72 @@
                   </dd>                  
 
               </div>
             </div>
             <div ng-switch-when="Docker">
-              <div ng-if="buildConfig.parameters.strategy.dockerStrategy.image">
+              <div ng-if="buildConfig.spec.strategy.dockerStrategy.image">
 
                   <dt>Builder image:</dt>
-                  <dd>{{buildConfig.parameters.strategy.dockerStrategy.image}}</dd>
+                  <dd>{{buildConfig.spec.strategy.dockerStrategy.image}}</dd>
 
               </div>
-              <div ng-if="buildConfig.parameters.strategy.dockerStrategy.from">
+              <div ng-if="buildConfig.spec.strategy.dockerStrategy.from">
                 <dl class="dl-horizontal">
                   <dt>Builder image stream:</dt>
-                  <dd>{{buildConfig.parameters.strategy.dockerStrategy.from | imageRepoReference : buildConfig.parameters.strategy.dockerStrategy.from.kind : buildConfig.parameters.strategy.dockerStrategy.Tag}}
+                  <dd>{{buildConfig.spec.strategy.dockerStrategy.from | imageObjectRef}}
                   </dd>                  
                 </dl>
               </div>
             </div>
             <div ng-switch-when="Custom">
-              <div ng-if="buildConfig.parameters.strategy.customStrategy.image">
+              <div ng-if="buildConfig.spec.strategy.customStrategy.image">
                 <dt>Builder image:</dt>
-                <dd>{{buildConfig.parameters.strategy.customStrategy.image}}</dd>
+                <dd>{{buildConfig.spec.strategy.customStrategy.image}}</dd>
               </div>
-              <div ng-if="buildConfig.parameters.strategy.customStrategy.from">
+              <div ng-if="buildConfig.spec.strategy.customStrategy.from">
                 <dt>Builder image stream:</dt>
-                <dd>{{buildConfig.parameters.strategy.customStrategy.from | imageRepoReference : buildConfig.parameters.strategy.customStrategy.from.kind : buildConfig.parameters.strategy.customStrategy.Tag}}
+                <dd>{{buildConfig.spec.strategy.customStrategy.from | imageObjectRef}}
                 </dd>                  
               </div>
             </div>
           </div>
-          <div ng-if="buildConfig.parameters.source">
-            <div ng-if="buildConfig.parameters.source.type == 'Git'">
+          <div ng-if="buildConfig.spec.source">
+            <div ng-if="buildConfig.spec.source.type == 'Git'">
 
                 <dt>Source repo:</dt>
-                <dd ng-bind-html='buildConfig.parameters.source.git.uri | githubLink | linky'></dd>
+                <dd ng-bind-html='buildConfig.spec.source.git.uri | githubLink | linky'></dd>
 
             </div>
           </div>
-          <div ng-if="buildConfig.parameters.output">
-            <div ng-switch="buildConfig.parameters.output.to.kind">
+          <div ng-if="buildConfig.spec.output">
+            <div ng-switch="buildConfig.spec.output.to.kind">
```

Comment:
actually, the `imageObjectRef` filter handles different kinds... think the whole output section should change to 
```
<div ng-if="buildConfig.spec.output.to">
  <dt>Output to:</dt>
  <dd>{{buildConfig.spec.output.to | imageObjectRef}}</dd>
</div>
```